addons/website_blog/static/src/css/website_blog.css

```diff
@@ -164,10 +172,39 @@ div#blog_angle_down a:hover {
   font-weight: normal;
 }
 
-mark + .popover {
+mark + .popover, .cover_footer{
   cursor: pointer;
 }
 
 .discussion_scroll_post {
   overflow-x: auto;
 }
+
+.blog_publish_btn {
+  padding-top: 16px;
+}
+
+.cover_footer_bg[style*="background-image: none;"] {
+  background-color: #B8B8B8;
+}
+
+#js_blogcover {
+  background-blend-mode: multiply;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 100%;
+}
+
+.o_none {
+  background-color: #FFFFFF;
+}
+.o_yellow {
+  background-color: #A2A51B;
+}
+.o_green {
+  background-color: #149F2C;
+}
```

Comment:
move those 3 colors to website.css (website module), around L770 with the others colors. Change the prefix to 'oe_'